src/Areas/AzureBestPractices/Commands/BestPracticesCommand.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Reflection;
+using AzureMcp.Areas.AzureBestPractices.Options;
+using AzureMcp.Commands;
+using AzureMcp.Helpers;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Areas.AzureBestPractices.Commands;
+
+public sealed class BestPracticesCommand(ILogger<BestPracticesCommand> logger) : BaseCommand
+{
+    private const string CommandTitle = "Get Azure Best Practices";
+    private readonly ILogger<BestPracticesCommand> _logger = logger;
+    private static readonly Dictionary<string, string> s_bestPracticesCache = new();
+
+    private readonly Option<string> _resourceOption = BestPracticesOptionDefinitions.Resource;
+    private readonly Option<string> _actionOption = BestPracticesOptionDefinitions.Action;
+
+    public override string Name => "get";
+
+    public override string Description =>
+        "Returns best practices for secure, production-grade Azure usage. " +
+        "Call this command and follow its guidance before generating or suggesting any Azure code. " +
+        "This command returns the content of the best practices file as a string array.";
+
+    public override string Title => CommandTitle;
+
+    protected override void RegisterOptions(Command command)
+    {
+        command.AddOption(_resourceOption);
+        command.AddOption(_actionOption);
+    }
+
+    [McpServerTool(Destructive = false, ReadOnly = true, Title = CommandTitle)]
+    public override Task<CommandResponse> ExecuteAsync(CommandContext context, ParseResult parseResult)
+    {
+        var resource = parseResult.GetValueForOption(_resourceOption);
+        var action = parseResult.GetValueForOption(_actionOption);
+
+        if (string.IsNullOrEmpty(resource) || string.IsNullOrEmpty(action))
+        {
+            context.Response.Status = 400;
+            context.Response.Message = "Both resource and action parameters are required.";
+            return Task.FromResult(context.Response);
+        }
```

Comment:
We could follow the validation logic in [`BaseAzureService.ValidateRequiredParameters()`](https://github.com/Azure/azure-mcp/blob/a9acbd337ab495444ca0620af8f951959a40389c/src/Services/Azure/BaseAzureService.cs#L138) for consistency in the error message.